src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,6 +1,29 @@
-// Don't forget to import the React library
+import React from 'react';
+import { PreparedTodo } from '../../types';
+import { UserInfo } from '../UserInfo';
 
-// Create a `TodoInfo` component accepting a `todo` object and use it in the
-// list to render `title`, `completed` status and `User`
+type Props = {
+  todo: PreparedTodo;
+};
 
-// Add a default export statement for TodoInfo component to use it in the other files
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
```

Comment:
since you have a type `PreparedTodo` - you can use it directly
and de-structure for further usage
```suggestion
export const TodoInfo: React.FC<PreparedTodo> = ({ title, completed, user }) => {
```